lib/sqlalchemy/orm/util.py

```diff
@@ -312,9 +313,8 @@ def identity_key(*args, **kwargs):
 
       E.g.::
 
-        >>> row = engine.execute(\
-            text("select * from table where a=1 and b=2")\
-            ).first()
+        >>> row = engine.execute(
+        ...     text("select * from table where a=1 and b=2")).first()
```

Comment:
Yes, CaselIT you are right. I try to fix case when there're a lot of spaces in example in documentation. See below:

![Screenshot from 2020-06-22 13-51-05](https://user-images.githubusercontent.com/5857672/85279738-99b01c00-b48f-11ea-9205-dc499e446689.png)

It happened because newline escaping was using in **not** raw-docstrings. The problem can be solved either add ``r`` before the opening quote  or delete ``\`` at end of line in docstrings. Now I think adding ``r`` would be preferable.

Reply:
The `r` sounds good

Reply:
@CaselIT I discard my changes and made raw-docstrings. See this [commit](https://github.com/sqlalchemy/sqlalchemy/pull/5371/commits/444a93cfa7e58c088d6b0fa6cbae35e65f371e5c)

Currently, this look like this:

![Screenshot from 2020-06-22 17-46-43](https://user-images.githubusercontent.com/5857672/85302344-b2c9c480-b4b1-11ea-97b7-9e80b9523e8a.png)